js/levels.js

```diff
@@ -29,6 +29,7 @@ function createFirstLevel()
     [door.handy, door.boat, door.river, desert.universe, door.cake, door.newYear],
     [door.both, door.square, door.quake, desert.universe, door.cake, door.mountains],
     [door.both, door.boat, door.river, desert.universe, door.oldRoom, door.mountains],  
+    [door.both, door.boat, door.river, desert.universe, door.oldRoom, door.CustomRoom],  
```

Comment:
That won't work

Reply:
So what should I do in this line

Reply:
Instead of adding a line, delete one wall used more than once and add yours.

Reply:
Sure thanks